read-fonts/src/lib.rs

```diff
@@ -37,6 +37,9 @@ pub use read::{ComputeSize, FontRead, FontReadWithArgs, ReadArgs, ReadError, Var
 pub use table_provider::TableProvider;
 pub use table_ref::TableRef;
 
+/// Public re-export of the font-types crate.
+pub use font_types as types;
```

Comment:
Purely out of curiosity, why are we changing the name (as types)?

Reply:
`read_fonts::font_types` is a mouthful and a bit redundant. It's fairly common to chop off the common parts when you're re-exporting within a family of crates.